app/js-src/agora.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import $ from "jquery"
+import CTZN from "./ctzn.js"
+import {state} from "./fission.js"
+import {initEditor} from "./editor.js"
+
+$(async () => {
+  // on page ready
+  console.log("page loaded")
+
+  $("#mini-cli").on("click", () => $("#mini-cli").val(""))
```

Comment:
not particular to this PR, but I've been wanting to disable this -- would you mind? it could be an optional setting.

one key use case for me is broken: I often want to edit the command here, for example prefix the node name with go/ or soon type a composition. in general it just feels surprising. we could also just have a small 'clear' button?